src/app/_utils/convertISOTime.ts

```diff
@@ -0,0 +1,46 @@
+type extractTimeFromISOType = {
+  day: number
+  month: string
+  time: string
+}
+
+export function extractTimeFromISO(
+  isoDateString: string,
+): extractTimeFromISOType {
+  const date = new Date(isoDateString)
```

Comment:
How do we check that this date is in the right ISO format? TypeScript only checks that a string is passed.

Maybe we should add an extra check to these 2 functions? What do you think? Maybe not critical for now.

Reply:
Update: The string passed comes from the Google API, so we can trust that it's safe. We can ignore this check for now.

But in general, I like to validate each piece of data that comes from outside our code :)